view/adminhtml/templates/grid.phtml

```diff
@@ -396,7 +402,7 @@ $uniqueId = '_' . uniqid();
 
                 getHiddenCols() {
                     const hiddenColsJson = window.sessionStorage.getItem(this.hiddenColsStorageName);
-                    return hiddenColsJson ? JSON.parse(hiddenColsJson) : [];
+                    return hiddenColsJson ? JSON.parse(hiddenColsJson) : JSON.parse(JSON.stringify(this.hiddenCols));
```

Comment:
why is `this.hiddenCols` encoded to json and decoded again?
Wouldn't it be enough do do
```js
return hiddenColsJson ? JSON.parse(hiddenColsJson) : this.hiddenCols;
```